css/css-overflow/parsing/overflow-valid.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Overflow: parsing overflow with valid values</title>
+<link rel="help" href="https://drafts.csswg.org/css-overflow/#propdef-overflow">
+<meta name="assert" content="overflow supports the full grammar '[ visible | hidden | clip | scroll | auto ]{1,2}'.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/support/parsing-testcommon.js"></script>
+</head>
+<body>
+<script>
+test_valid_value("overflow", 'visible');
+test_valid_value("overflow", 'hidden');
+test_valid_value("overflow", 'clip');
+test_valid_value("overflow", 'scroll');
+test_valid_value("overflow", 'auto');
+
+test_valid_value("overflow", 'visible visible', 'visible');
+test_valid_value("overflow", 'hidden visible');
+test_valid_value("overflow", 'clip clip', 'clip');
+test_valid_value("overflow", 'scroll auto');
+test_valid_value("overflow", 'auto auto', 'auto');
+
```

Comment:
This is all correct, but you should also check this sentence:

> Computed value: as specified, except with visible/clip computing to auto/hidden (respectively) if one of overflow-x or overflow-y is neither visible nor clip


```suggestion
test_valid_value("overflow", 'auto visible', 'auto');
test_valid_value("overflow", 'visible auto', 'auto');
test_valid_value("overflow", 'hidden visible', 'hidden auto');
test_valid_value("overflow", 'visible hidden', 'auto hidden');
test_valid_value("overflow", 'scroll visible', 'scroll auto');
test_valid_value("overflow", 'visible scroll', 'auto scroll');

test_valid_value("overflow", 'auto clip', 'auto hidden');
test_valid_value("overflow", 'clip auto', 'hidden auto');
test_valid_value("overflow", 'hidden clip', 'hidden');
test_valid_value("overflow", 'clip hidden', 'hidden');
test_valid_value("overflow", 'scroll clip', 'scroll hidden');
test_valid_value("overflow", 'clip scroll', 'hidden scroll');
```

Reply:
You're right, my bad. Yes, please add this in the computed value TC.

Reply:
Computed value tests: https://github.com/web-platform-tests/wpt/pull/14923